bundle/manifests/kuadrant-operator.clusterserviceversion.yaml

```diff
@@ -256,37 +244,68 @@ spec:
                   periodSeconds: 10
                 resources:
                   limits:
-                    cpu: 200m
-                    memory: 100Mi
+                    cpu: 100m
+                    memory: 30Mi
                   requests:
                     cpu: 100m
                     memory: 20Mi
                 securityContext:
                   allowPrivilegeEscalation: false
+                volumeMounts:
+                - mountPath: /controller_manager_config.yaml
+                  name: manager-config
+                  subPath: controller_manager_config.yaml
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=10
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
I would remove the rbac proxy from the deployment. Maybe in other PR.

We removed it from the 3scale operator 3scale/3scale-operator#692

3scale Ops team has also removed the rbac proxy from their operator 3scale-ops/prometheus-exporter-operator#26

Check out for the reasons in the PRs

Reply:
Created follow up JIRA for this https://issues.redhat.com/browse/KUADRANT-32